.tool-versions

```diff
@@ -0,0 +1 @@
+elixir 1.6.1
```

Comment:
This should not be commited since we can’t assume what the developer will use to manage elixir and erlang dependancies.